### DISCUSSION TITLE
How to know if Hardware Encoding is used (Quicksync) & if relay is used or direct connection (Windows)

### DISCUSSION THREAD

**Discussion by Hawxxer** [rustdesk/rustdesk#5079] [Q&A]
Maybe someone can hint me to this, I guess it should be stated somewhere in a log but my docker Server and Relay does not give me a hint if it is directly connected or relayed.
So my Server and Relay is a Docker but my Host and Client are both Windows PCs. Can you tell me where I find that information?

the folder C:\Windows\ServiceProfiles\LocalService\AppData\Roaming\RustDesk\log\server\ is not found but I guess this would be the case if I installed the Server not just the host/client application

**Comment by 21pages** [comment]
hardware encoding: H264/H265.
relay or direct: there is an icon/toolltip on remote window tab.
